zeus/optimizer/batch_size/server/config.py

```diff
@@ -0,0 +1,33 @@
+"""App setting is read here."""
+
+from pydantic.env_settings import BaseSettings
+from dotenv import find_dotenv
+
+
+class ZeusBsoSettings(BaseSettings):
+    """App setting.
+
+    Attributes:
+        database_url: url of database for the server
+        database_password: password of database if any
+        echo_sql: log sql statements it executes
+        log_level: level of log
+    """
+
+    database_url: str
+    database_password: str = ""
+    echo_sql: bool = True
+    log_level: str = "DEBUG"
```

Comment:
https://github.com/ml-energy/zeus/blob/f9275786fd98920b3bf98485c78f9e15b2304364/zeus/util/logging.py#L55
`settings.log_level` should override `ZEUS_LOG_LEVEL` IMO. `ZEUS_LOG_LEVEL` is like a global default and users should be able to set a more specific (finer-grained) variable (`--log-level` in the BSO server command line) or `ZEUS_BSO_LOG_LEVEL` to override the framework-level default.

Reply:
Oh.. I thought https://github.com/show981111/zeus/blob/master/zeus/optimizer/batch_size/server/router.py#L26 this is going to override everything, but seems like calling a `get_logger` will override the log level again to the `ZEUS_LOG_LEVEL` ?

Reply:
Hmmmm actually `get_logger` will override again. I'll think about this fix it later. Let's leave things as is for now.